cli/mender-artifact/main.go

```diff
@@ -56,6 +56,18 @@ func getCliContext() *cli.App {
 	app.Author = "mender.io"
 	app.Email = "contact@mender.io"
 
+	privateKeyFlag := cli.StringFlag{
+		Name: "key, k",
+		Usage: "Full path to the public key that will be used to verify " +
```

Comment:
```suggestion
		Usage: "Full path to the private key that will be used to verify " +
```

Reply:
Hehehe, good catch :)

I will fix it in PR #194 where the commit belongs.